core/amber/src/test/scala/edu/uci/ics/amber/engine/architecture/worker/WorkerSpec.scala

```diff
@@ -100,7 +104,7 @@ class WorkerSpec
     val invocation = ControlInvocation(0, AddPartitioning(mockTag, mockPolicy))
     worker ! NetworkMessage(
       0,
-      WorkflowControlMessage(CONTROLLER, 0, invocation)
+      WorkflowControlMessage(ChannelEndpointID(CONTROLLER, true), 0, invocation)
```

Comment:
please add keyword to those arguments when it is not straightforward what it is, e.g., `ChannelEndpointID(CONTROLLER, isControl=true)`.